server/config.js

```diff
@@ -95,27 +104,100 @@ module.exports = {
     showStackTraces:
       getEnv('ENABLE_STACK_TRACES_ON_ERROR_PAGES', 'false') === 'true',
     useRedisCache: getEnv('ENABLE_REDIS_CACHE', 'true') === 'true',
-    approvedVisitorsFeatureEnabled:
-      getEnv('APPROVED_VISITORS_FEATURE_ENABLED', 'false') === 'true',
-    adjudicationsFeatureEnabled:
-      getEnv('ADJUDICATIONS_FEATURE_ENABLED', 'false') === 'true',
-    adjudicationsFeatureEnabledAt: [
-      'cookhamwood',
-      'erlestoke',
-      'felthama',
-      'felthamb',
-      'garth',
-      'lindholme',
-      'newhall',
-      'ranby',
-      'stokeheath',
-      'styal',
-      'swaleside',
-      'themount',
-      'wayland',
-      'werrington',
-      'wetherby',
-    ],
+  },
+  sites: {
+    berwyn: {
+      enabled: false,
+      features: [],
+    },
+    bullingdon: {
+      enabled: true,
+      features: [
+        'approvedVisitors',
+        'incentives',
+        'money',
+        'timetable',
+        'visits',
+      ],
+    },
+    chelmsford: {
+      enabled: true,
+      features: ['incentives', 'money', 'timetable', 'visits'],
```

Comment:
Minor nit - I'd suggest using multiline arrays consistently for the features. I guess this isn't multiline because the code line length doesn't mandate it, but it's easier to spot/review changes when they are multiline. And consistency across establishments makes visual comparisons easier.

Reply:
Unfortunately this is due to code formatting and the linter, so I can put it back and then the linter will just reformat according to it's rules.